package.json

```diff
@@ -22,13 +24,13 @@
   },
   "homepage": "https://github.com/mikeal/dag-json#readme",
   "devDependencies": {
-    "aegir": "^21.3.0",
     "hundreds": "0.0.2",
-    "tsame": "^2.0.1"
+    "mocha": "^7.1.2",
```

Comment:
As you know I like the idea of having a uniform tool for maintenance work, hence I prefer having Aegir. It makes releases easy and also creates a changelog for the releases, which is very handy for breaking changes (if you use the right keywords in your commit message).

You are the maintainer, so it's your call. Having a proper changelog for releases is a good idea though IMHO.

Reply:
I’m automating all of the releases with `merge-release`. This setup is working quite well for other projects, and given the recent security issue in `aegir publish` I’m quite happy with my decision not to use it.

Reply:
Though now changelog file or changes listed on the GitHub release page. I think that would be super helpful (especially with breaking changes).